spring2/src/main/java/com/bates/spring/UserController.java

```diff
@@ -0,0 +1,73 @@
+package com.bates.spring;
+
+
+import com.bates.spring.dto.UserDto;
+import main.java.com.bates.spring.model.Users;
+import main.java.com.bates.spring.model.request.UserRequest;
+import main.java.com.bates.spring.model.response.response.UserResponse;
+import com.bates.spring.services.UserService;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.beans.BeanUtils;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("users")
+public class UserController {
+
+    private final UserService userservice;
+
+    public UserController(UserService userservice){
+        this.userservice = userservice;
+    }
+
+    @GetMapping(path = "/allUsers")
+    public List<UserResponse> getAllUsers(){
+        List<UserResponse> returnValue = userservice.getAllUsers();
+        return returnValue;
+    }
+
+    @GetMapping
+    public List<UserDto> getUsers(@RequestParam(value = "page", defaultValue = "1") int page,
```

Comment:
Oh now I see why you changed the path...but you could put that logic here and set the defaults to 0, check if they're 0 and if so, follow the get all logic. That way your endpoints are still restful